install/0000_00_cluster-version-operator_03_deployment.yaml

```diff
@@ -23,6 +23,7 @@ spec:
           - "start"
           - "--release-image={{.ReleaseImage}}"
           - "--enable-auto-update=false"
+          - "--enable-default-cluster-version={{.Bootstrap|not}}"
```

Comment:
don't change this... rather the https://github.com/openshift/cluster-version-operator/blob/master/bootstrap/bootstrap-pod.yaml

Reply:
> don't change this... rather the https://github.com/openshift/cluster-version-operator/blob/master/bootstrap/bootstrap-pod.yaml

[Done](https://github.com/openshift/cluster-version-operator/pull/238#issuecomment-523274889).